experiments/enableTTL/README.md

```diff
@@ -0,0 +1,188 @@
+# Effect of `enableTTL`
+
+Dispatcher versions 4.1.11 and greater can be configured to respect a time-to-live (TTL) based content "timeout". TTL headers (`Cache-Control` or `Expires`) must be included in responses sent from the publish tier in order for the dispatcher to know how long a cached file should "live" for in the cache. When this configuration is enabled (and reasonable TTL values are set), there is no longer a need for a dispatcher flush agent.
```

Comment:
I don't think you need the word "for" here after "live".   i.e.  "...file should live in the cache"